### PR TITLE
Decrease hero height and remove background image

### PR DESCRIPTION
**Pull request by nielsvanvelzen**
image

### CODE REVIEW

File: package-lock.json [hunk 1]
```diff
@@ -1,8 +1,12971 @@
 {
   "name": "jellyfin-org",
   "version": "0.0.0",
-  "lockfileVersion": 1,
+  "lockfileVersion": 2,
```
Comment (MrTimscampi) [inline package-lock.json:4]: We probably want to keep Node 14/NPM 6 until Node 16 enters LTS (It's not there yet, I think it's for next month).

**Review by MrTimscampi — COMMENTED** [review]
If we're going to replace the image later, shouldn't we simply keep the current one until then and only reduce the size?

**Comment by nielsvanvelzen** [comment]
> If we're going to replace the image later, shouldn't we simply keep the current one until then and only reduce the size?

The image looked really ugly with this size (pretty much only the television is visible) so I just removed it.

File: package-lock.json (hunk 1 above)
Comment (ferferga) [reply package-lock.json]: On October 26

https://nodejs.org/en/about/releases/

**Review by MrTimscampi — APPROVED** [review]

**Pull request merged**